Source/WebCore/platform/graphics/GraphicsContextGL.h

```diff
@@ -1450,6 +1451,9 @@ class GraphicsContextGL : public RefCounted<GraphicsContextGL> {
 
     virtual void setContextVisibility(bool) = 0;
 
+    virtual DestinationColorSpace getDrawingBufferColorSpace();
+    virtual void setDrawingBufferColorSpace(const DestinationColorSpace&);
```

Comment:
To be consistent, you might still mark this up with `WEBCORE_EXPORT`.
For clang, the symbol must be exported when an implementation from other library does not override the base implementation. Since all WebKit library implementations override this, it's not a problem for clang.
However,
IIRC there was might have been some related problems for other compilers (gcc or msvc) where the symbol rules might be a bit different.